src/transports/dash/manifest_parser.ts

```diff
@@ -180,7 +180,7 @@ export default function generateManifestParser(
           });
           return processMpdParserResponse(value.continue(resources));
         }
-      }).catch((err) => {
+      }, (err) => {
         const error = formatError(err, {
```

Comment:
Also I wonder if formatting the error here is needed. At worst scenario, it will already be formatted by the ManifestFetcher (`src/core/fetchers/manifest/manifest_fetcher.ts`)